argo-cd-apps/app-of-app-sets/base/all-application-sets.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   project: default
   source:
-    path: argo-cd-apps/environments/staging
+    path: argo-cd-apps/environments/production
```

Comment:
Sure, will add for production as well.

Reply:
I'm not sure if we want 2 environments here since @gbenhaim added staging and production in different places and it's not working when deployed, I have modified it to production.

Reply:
we need to have production and stating overlays.

> @gbenhaim added staging and production in different places and it's not working when deployed

What is the error you encountered ?

Reply:
Hi Gal, Getting below error 
`Failed to load target state: failed to generate manifest for source 1 of 1: rpc error: code = Unknown desc = components/rbac/staging/: app path does not exist`

Reply:
@gbenhaim @eisraeli I have added config for staging as well.